tests/gatsby-2/gatsby-config.js

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
All of these `tests/gatsby-x/` directories were seeded from the [`hello-world`](https://github.com/gatsbyjs/gatsby-starter-hello-world) starter. I just checked out the code at the commits that marked the highest of each major version, and run the starter against that local code, and voila.